gmailRest/gmailRest-connector/gmailRest-connector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/GmailSMTPConnectionObject.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.carbon.connector;
+
+import com.google.code.com.sun.mail.smtp.SMTPTransport;
+import com.google.code.javax.mail.Session;
+
+/**
+ * This class provides the object model to store {@link com.google.code.javax.mail.Session} and
+ * {@link com.google.code.com.sun.mail.smtp.SMTPTransport} instances to send mails through SMTP.
+ */
+public class GmailSMTPConnectionObject {
```

Comment:
Classes are considered "blue prints" of objects. So, naming a class with the notion of an "object" is inappropriate.